inference-engine/src/mkldnn_plugin/memory_desc/cpu_memory_desc_utils.cpp

```diff
@@ -69,20 +89,27 @@ BlockedMemoryDescPtr MemoryDescUtils::convertToBlockedMemoryDesc(const MemoryDes
     }
 }
 
-InferenceEngine::Blob::Ptr MemoryDescUtils::interpretAsBlob(const MKLDNNMemory &mem) {
+InferenceEngine::Blob::Ptr MemoryDescUtils::interpretAsBlob(const MKLDNNMemory &mem, bool withDefaulStrides) {
     // TODO [DS]: Rewrite when IE is moved to the new TensorDescriptor
     auto& memDesc = mem.getDesc();
-    InferenceEngine::TensorDesc desc = convertToTensorDesc(memDesc);
+    InferenceEngine::TensorDesc desc = convertToTensorDesc(memDesc, withDefaulStrides);
 
     desc = InferenceEngine::TensorDesc(desc.getPrecision(), memDesc.getShape().getStaticDims(), desc.getBlockingDesc());
     return make_blob_with_precision(desc, mem.GetData());
 }
 
-InferenceEngine::TensorDesc MemoryDescUtils::convertToTensorDesc(const MemoryDesc& desc) {
+InferenceEngine::TensorDesc MemoryDescUtils::convertToTensorDesc(const MemoryDesc& desc, bool withDefaulStrides) {
```

Comment:
Honestly I'm not sure that this flag is legal. If we have strided tensor in the plugin, we have to convert its descriptor using the same strides since we will use the same underlying memory. So this is a source of hard-to-find errors when developer can set true by mistake.

Reply:
Since we need this WA only for case when we have empty tensor I will revert this changes. Now we convert desc to tensor desc disregarding strides only in empty tensor case.